recipe/meta.yaml

```diff
@@ -1,17 +1,23 @@
 {% set name = "libsndfile" %}
-{% set version = "1.0.31" %}
+{% set version = "1.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/libsndfile/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.bz2
-  sha256: a8cfb1c09ea6e90eff4ca87322d4168cdbe5035cb48717b40bf77e751cc02163
+  # libsndfile 1.1.0 shipped with slightly broken package archives
+  # these should be reconciled into one download in a subsequent release
+  fn: {{ name }}-{{ version }}.zip  # [win]
+  url: https://github.com/libsndfile/{{ name }}/archive/refs/tags/{{ version }}.zip  # [win]
+  sha256: d71292106e6f75c3b6dc0292087d935507a34c3b93ad65c00d3d5729c5eecad2  # [win]
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
```

Comment:
there is a slight weirdness going on here -- the fn says `tar.gz` but the URL says `tar.xz`. 
You can just completely remove `fn` from both downloads, things will work fine.

Reply:
Thanks for catching that - resolved